plugins/org.eclipse.fordiac.ide.model.commands/src/org/eclipse/fordiac/ide/model/commands/change/AbstractUpdateFBNElementCommand.java

```diff
@@ -513,4 +521,9 @@ public FBNetworkElement getNewElement() {
 	public Set<EObject> getAffectedObjects() {
 		return Set.of(network);
 	}
+
+	private static boolean isInternal(final Attribute attribute) {
```

Comment:
i think this method would also be usefull for others.
Please rename it to **isInternalAttribute** and move it to `org.eclipse.fordiac.ide.model.datatype.helper.InternalAttributeDeclarations`